apps/rush-lib/src/cli/actions/LinkAction.ts

```diff
@@ -33,6 +33,7 @@ export class LinkAction extends BaseRushAction {
   }
 
   protected run(): Promise<void> {
+    this.runRushCheckIfNecessary();
```

Comment:
>this.runRushCheckIfNecessary(); [](start = 4, length = 31)

This seems like a weird place to perform the check

Reply:
I don't think so... Why would I do the check after getLinkManager?

---
In reply to: [218977986](https://github.com/Microsoft/web-build-tools/pull/829#discussion_r218977986) [](ancestors = 218977986)

Reply:
"rush link" runs only after a successful "rush install", so there doesn't seem to be any point to enforce this here.

---
In reply to: [218981572](https://github.com/Microsoft/web-build-tools/pull/829#discussion_r218981572) [](ancestors = 218981572,218977986)